python/lsst/meas/astrom/pessimistic_pattern_matcher_b_3D.py

```diff
@@ -123,11 +123,11 @@ class we use id to reference the position in the input reference
             self._pair_id_array[ref_id, ref_id:] = sub_id_array[:, 1]
             self._pair_delta_array[ref_id, ref_id:, :] = sub_delta_array
             self._pair_dist_array[ref_id, ref_id:] = sub_dist_array
-
+            
             # Don't fill the array column wise if we are on the last object
             # to avoid array overrun.
```

Comment:
Could you clarify this comment? I think you fill the array if we are not on the last object, but if you are on the last object then you don't fill the array at all. If that's really what you are doing then something like this might be clearer:

    # If not the last object, append ?? to the _pair arrays. Otherwise do...??? to avoid array overrun